### ISSUE TITLE
Finder Library

### ISSUE THREAD

**Issue by akasa-denada** [SED-EEW/SED-EEW-SeisComP-contributions#57]
The link for [FinDer](https://github.com/SED-EEW/FinDer) library is broken. Can you fix this?

**Comment by FMassin** [comment]
No, the code for FinDer is private, sorry. Please see the link to documentation to know more?
